relay-server/src/extractors/request_meta.rs

```diff
@@ -571,3 +571,71 @@ impl FromRequest<ServiceState> for EnvelopeMeta {
         Ok(Box::new(future))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    impl RequestMeta {
+        // TODO: Remove Dsn here?
+        pub fn new(dsn: relay_common::Dsn) -> Self {
+            Self {
+                dsn: PartialDsn::from_dsn(dsn).expect("invalid DSN"),
+                client: Some("sentry/client".to_string()),
+                version: 7,
+                origin: Some("http://origin/".parse().unwrap()),
+                remote_addr: Some("192.168.0.1".parse().unwrap()),
+                forwarded_for: String::new(),
+                user_agent: Some("sentry/agent".to_string()),
+                no_cache: false,
+                start_time: Instant::now(),
+                client_hints: ClientHints::default(),
```

Comment:
Could you please add tests which use non-default `client_hints`? To find out what tests would be affected, change to a non-default value here and see if any tests break.